github/handlers/HandleInvalidRepoName.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    IHttp,
+    IModify,
+    IRead,
+} from "@rocket.chat/apps-engine/definition/accessors";
+import { GithubApp } from "../GithubApp";
+import { isRepositoryExist } from "../helpers/githubSDK";
+import { getAccessTokenForUser } from "../persistance/auth";
+import { IUser } from "@rocket.chat/apps-engine/definition/users";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { IAuthData } from "@rocket.chat/apps-engine/definition/oauth2/IOAuth2";
+
+export async function HandleInvalidRepoName(
+    repoName: string,
+    http: IHttp,
+    app: GithubApp,
+    modify: IModify,
+    sender: IUser,
+    read: IRead,
+    room: IRoom
+): Promise<boolean> {
+    const accessTokenInfo = (await getAccessTokenForUser(
```

Comment:
@Nabhag8848 Although the variable name `accessTokenInfo` makes sense, we have uses `accessToken` for this data throughout the code. Keeping the naming patterns consistent will help future contributors as well.